samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/setup-controller.sh

```diff
@@ -0,0 +1,299 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root"
+  exit
+fi
+
+# This is a script to create single-node Kubernetes cluster and deploy Azure Arc Data Controller on it.
+#
+export AZUREARCDATACONTROLLER_DIR=aadatacontroller
+
+# Get password as input. It is used as default for controller, SQL Server Master instance (sa account).
+#
+while true; do
+    read -s -p "Create Password for Azure Arc Data Controller: " password
+    echo
+    read -s -p "Confirm your Password: " password2
+    echo
+    [ "$password" = "$password2" ] && break
+    echo "Password mismatch. Please try again."
+done
+
+# Name of virtualenv variable used.
+#
+export LOG_FILE="aadatacontroller.log"
+export DEBIAN_FRONTEND=noninteractive
+
+# Requirements file.
+export AZDATA_PRIVATE_PREVIEW_DEB_PACKAGE="https://aka.ms/aadatacontrollerazdata"
+
+# Kube version.
+#
+KUBE_DPKG_VERSION=1.15.0-00
+KUBE_VERSION=1.15.0
+
+# Wait for 5 minutes for the cluster to be ready.
+#
+TIMEOUT=600
+RETRY_INTERVAL=5
+
+# Github related contstants
+#
+GITHUB_AADATACONTROLLER_BRANCH=master
+
+# Variables used for azdata cluster creation.
+#
+export AZDATA_USERNAME=admin
```

Comment:
Are these needed?

Reply:
Not sure about ### AZDATA_USERNAME and ### AZDATA_PASSWORD but the rest are. Will double check if deployment works without these two and if successful will send another iteration.

Reply:
Please create a top-level folder called "azure-arc" and put the deployment files there (ex: samples\features\azure-arc, samples\features\azure-arc\deployment). This way in future other samples can be added under the same parent directory.